build.gradle

```diff
@@ -231,6 +231,7 @@ project(':iceberg-core') {
       exclude group: 'org.slf4j', module: 'slf4j-log4j12'
     }
 
+    testCompile "com.h2database:h2"
```

Comment:
Should this be `testRuntime` because the only compile dependency is for JDBC?

Why switch databases?

Reply:
updated, wasn't sure about license compatibility, H2 license looked better.

Reply:
The MPL license is a copyleft license, which means that in principle people are not allowed to distribute code that is under the MPL-2.0 license under different terms. The GPL licenses (including LGPL and AGPL) require that the entire application is distributed under the terms of the GPL license.

I think we cannot use it. Could you switch to use https://mvnrepository.com/artifact/org.xerial/sqlite-jdbc?